examples/create-fixture-extension.test.ts

```diff
@@ -34,9 +37,12 @@ const fixture = new Fixture({ seed: 38 }).extend([
 	totalVisitsGenerator,
 ]);
 const person = fixture.from(PersonSchema);
+// #endregion example
 
 test('generates a person', () => {
-	expect(person).toMatchInlineSnapshot(`
+	expect(person).toMatchInlineSnapshot(
```

Comment:
To make this prettier in the readme file I would do this differently.
Instead of user a snapshot, we can do:

```
const expected = 
  // #region output
  {
    /* output comes here */
  }
  // #endregion output

expect(person).toEqual(expected);
```

This way the addition ` tokens are not added to the snippets, and they are formatted better.

Reply:
Ha, smart :D
I changed the prettier settings for this 😅

Reply:
That was my first thought as well but I was like, "Tim probably doesn't want to change the prettier settings globally for this one little annoyance." 😂

Reply:
I feel pretty good about where this is at. I don't have full support for the Vitepress snippet syntax in my regex and I'm sure we'll want to add more Vitepress blocks/etc at some point, but I think this gets us a reasonable level of automation so that we only have to update the index.md.

I'd eventually like to write an actual lexer/parser but.... that's a bit more involved than I'd like to get right now 😅